### PR TITLE
Development onav 0.11.0

### PR DESCRIPTION
**Pull request by dobrienCP**

### CODE REVIEW

File: docs_outdoornav_user_manual/release_notes.mdx [hunk 1]
```diff
@@ -6,6 +6,30 @@ toc_min_heading_level: 2
...
+-   Stack Light can now be manually overridden/muted
+
+### Bug Fixes
+
+-   1688: Move PTZ task only ever uses the first PTZ camera's positions
+
+### Known Issues
+
+-   1396: Robot gets temporarily stuck at start of mission when near a Waypoint
+-   1844: [MPC] Maximum acceleration param doesn't appear to have any effect
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/release_notes.mdx:33]: Just noting that as of the meeting this morning that @jmastrangelo-cpr was still hoping to fix 1844. Just a note to remove the line here if it gets fixed.

File: docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/user_manual_husky_observer.mdx [hunk 2]
```diff
@@ -850,6 +860,11 @@ enable button (either slow or turbo mode) and use the left thumbstick to drive H
   </figure>
 </center>
 
+Both the PTZ camera and the driving lights can also be controlled through the controller. While holding the 
+X button the left thumbstick controls the Pan and Tilt of the camera. While holding the O button the left thumbstick 
+controls the zoom. The D-Pad is used to control the driving lights, with the top button controlling the front driving 
+lights and the bottom button controlling the read driving lights.
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/user_manual_husky_observer.mdx]: read -> rear

**Review by jhiggins-cpr — COMMENTED** [review]
Just a couple minor notes. Otherwise looks good.

File: docs_outdoornav_user_manual/release_notes.mdx (hunk 1 above)
Comment (dobrienCP) [reply docs_outdoornav_user_manual/release_notes.mdx:33]: Sounds good. I'll update this by end of day today if it's fixed and if not I'll leave it as is.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**